static/js/notificationsCenter.js

```diff
@@ -148,10 +148,15 @@ function loadMonitoringData(data) {
   }
 
   for (let i = 0; i < data.length; i++) {
+    var eventThreshold = data[i].EventThreshold
+    if (data[i].EventName === "monitoring_hdd_almostfull") {
+      eventThreshold = 1 - eventThreshold
+    }
```

Comment:
**Minor:** Please add a comment to this block as this is very weird :)

Reply:
Done.